std/fmt/index.zig

```diff
@@ -857,6 +940,157 @@ test "fmt.format" {
         const value: u8 = 'a';
         try testFmt("u8: a\n", "u8: {c}\n", value);
     }
+    {
+        const value: [3]u8 = "abc";
+        try testFmt("const [3]u8: abc\n", "const [3]u8: {}\n", value);
+    }
+    {
+        var value = "abc";
+        try testFmt("var []u8: abc\n", "var []u8: {}\n", value);
+    }
+    {
```

Comment:
I don't understand, I added these just because there weren't tests of quoted strings so I thought I'd add a couple. Could you provide a more information?

Reply:
You're only checking that the prefix is correct, because pointers have non-deterministic values at runtime. But you could use `@intToPtr(*T, 0xdeadbeef)` and then you can look for `"T@0xdeadbeef"` using `testFmt`.

Reply:
Got it, another option is to use @ptr2Int using "real" addresses:
```
    {
        var buf1: [32]u8 = undefined;
        var buf2: [32]u8 = undefined;
        const value = "abc";

        // Print Pointer Path3
        const expected = try bufPrint(buf2[0..], "&value: u8@{x}", @ptrToInt(&value));
        const actual = try bufPrint(buf1[0..], "&value: {p}", &value);
        wink_log_strln("expected=", expected); 
        wink_log_strln("actual=", actual); 
        try testExpectedActual(expected, actual);
    }

fn testExpectedActual(expected: []const u8, actual: []const u8) !void {
    if (mem.eql(u8, expected, actual)) return;

    std.debug.warn("\n====== expected this output: =========\n");
    std.debug.warn("{}", expected);
    std.debug.warn("\n======== instead found this: =========\n");
    std.debug.warn("{}", actual);
    std.debug.warn("\n======================================\n");
    return error.TestFailed;
}
```
 I like using "real" addresses where we can, do you have a preference?